db/migrate/20160926083804_create_donors.rb

```diff
@@ -0,0 +1,13 @@
+class CreateDonors < ActiveRecord::Migration
+  def change
+    create_table :donors do |t|
+      t.string :identification
```

Comment:
Identification should be mandatory, so let's validate that both in the model and here.